docs/source/api/v4/sslkey_expirations.rst

```diff
@@ -0,0 +1,93 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-sslkey_expirations:
+
+**********************
+``sslkey_expirations``
+**********************
+
+``GET``
+=======
+Retrieves SSL certificate expiration information.
+
+:Auth. Required: Yes
+:Roles Required: "admin"
+:Permissions Required: ACME:READ
```

Comment:
This reads more than just ACME keys right? Besides, right now, ACME:READ is used to express the ability to read ACME account details. The ability to read SSL keys is behind DS-SECURITY-KEY:READ. If this is supposed to be a separate endpoint _because_ we want to separate the ability to read expirations from the ability to read keys, then this should probably have a new Permission entirely to encapsulate that.